cmd/extensions/main.go

```diff
@@ -200,30 +173,14 @@ func main() {
 	// which is a requirements of Stackdriver, otherwise most of time series would be invalid for Stackdriver
 	metrics.SetReportingPeriod(ctlConf.PrometheusMetrics, ctlConf.Stackdriver)
 
-	// Add metrics controller only if we configure one of metrics exporters
-	if ctlConf.PrometheusMetrics || ctlConf.Stackdriver {
-		rs = append(rs, metrics.NewController(kubeClient, agonesClient, kubeInformerFactory, agonesInformerFactory))
```

Comment:
I'm not super familiar with metrics but it seems odd that we have none on the extensions side? Might be true.

Actually, I think this is an artifact of missing the allocation server, below.

Reply:
I think I'm still confused on how the metrics controller is used, and specifically how it's used by the game allocation server. I don't see it being explicitly passed to gamerserverallocations and I can't seem to connect the allocation events to the metrics.

Reply:
Removed everything thats related to metrics controller.